Development/DCM/DCMUserAccountManager/duam.py

```diff
@@ -87,12 +87,14 @@ def signInUser(self, p_loginData):
             passwordValid = verify_password(userData.getPassword(), password)
             # Redundant check to make sure correct user data is being returned
             usernameValid = (userData.getUsername() == username)
-
+            print(userData.getUsername() + userData.getPassword())
+            print(passwordValid)
```

Comment:
what's the point of these? if debugging, it's a good idea to put a label on it - print("passwordValid:", passwordValid)